builtin/k8s/releaser.go

```diff
@@ -65,6 +67,13 @@ func (r *Releaser) resourceManager(log hclog.Logger, dcr *component.DeclaredReso
 			resource.WithDestroy(r.resourceServiceDestroy),
 			resource.WithStatus(r.resourceServiceStatus),
 		)),
+		resource.WithResource(resource.NewResource(
+			resource.WithName("ingress"),
+			resource.WithState(&Resource_Ingress{}),
+			resource.WithCreate(r.resourceIngressCreate),
+			resource.WithDestroy(r.resourceIngressDestroy),
```

Comment:
Does this mean that when we destroy one deployment, it'll destroy the whole ingress resource that's in use by all deployments??

Reply:
Nope, it should stick around the same as the Service does that gets created on release. Good question tho!